background/ncn_kdump.md

```diff
@@ -87,6 +87,16 @@ to be installed; the `crash` command can not thoroughly analyze a dump without t
 
     * Install from Artifactory if credentials are available.
 
+        1. This step should ONLY be performed if an http proxy is required to access a public endpoint on the internet for the purpose of downloading artifacts.
+        CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
+        The http proxy variables must be `unset` after the desired artifacts are downloaded. Failure to unset the http proxy variables after downloading artifacts will cause many failures in subsequent steps.
+
+           ```bash
+           Example:
```

Comment:
Move the `Example:` out of the code snippet, or comment it out. Otherwise folks using the copy-paste button will copy the `Example`.